tests/e2e_test/test_bench.py

```diff
@@ -91,35 +62,57 @@ def get_markdown_data(key: str, head_name: str):
 @pytest.mark.asyncio
 @pytest.mark.skipif(torch.cuda.device_count() < 1, reason="at least 1 gpus required for simple benchmark")
 @pytest.mark.parametrize("model", ['/mnt/model/Qwen-7B'])
-async def test_simple_benchmark(setup_ray_env, model):
+async def test_simple_benchmark(cleanup_ray_env, shutdown_llumnix_service, model):
     device_count = torch.cuda.device_count()
+    ip = "127.0.0.1"
     base_port = 37037
+    ip_ports = []
     for i in range(device_count):
+        port = base_port+i
+        ip_port = f"{ip}:{port}"
+        ip_ports.append(ip_port)
         launch_command = generate_launch_command(result_filename=str(base_port+i)+".out",
-                                                 launch_ray_cluster=False, port=base_port+i, model=model)
+                                                 launch_ray_cluster=False,
+                                                 ip=ip,
+                                                 port=port,
+                                                 model=model)
         subprocess.run(launch_command, shell=True, check=True)
 
-    await asyncio.sleep(60)
+    wait_for_llumnix_service_ready(ip_ports)
 
-    async def run_bench_command(command):
-        process = await asyncio.create_subprocess_shell(command)
-        await process.wait()
-        assert process.returncode == 0
+    def run_bench_command(command):
+        # pylint: disable=consider-using-with
+        process = subprocess.Popen(command, shell=True)
+        return process
 
     tasks = []
     for i in range(device_count):
-        bench_command = generate_bench_command(ip_ports=f"127.0.0.1:{base_port+i}", model=model, num_prompts=300,
-                                               dataset_type="sharegpt",
-                                               dataset_path="/mnt/dataset/sharegpt_gpt4/sharegpt_gpt4.jsonl" ,
-                                               qps=2,
-                                               results_filename=f"{base_port+i}.out")
-        tasks.append(run_bench_command(bench_command))
-
-    await asyncio.wait(tasks, timeout=60*30)
+        bench_command = generate_bench_command(
+            ip_ports=f"127.0.0.1:{base_port + i}",
+            model=model,
+            num_prompts=200,
```

Comment:
200 num_prompts is too few

Reply:
You had changed it to 200. I think 200 is enough for bench test, and we can make migration test more strict because it has more checks than bench test.